src/components/ActionsMenu/ActionsMenu.st.css

```diff
@@ -0,0 +1,49 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, fallback;
+}
+
+:vars {
+    /*Defaults*/
+    DefaultMenuBackgroundColor: color-1;
+    DefaultMenuBorderWidth: 1px;
+    DefaultMenuBorderColor: rgba(255, 255, 255, 0.4);
+}
+
+:vars {
+    /*Overrides*/
+    MenuBackgroundColor:  --overridable;
+    MenuBorderWidth:  --overridable;
+    MenuBorderColor:  --overridable;
+}
+
+:vars {
+    menuBackgroundColor: color(fallback(value(MenuBackgroundColor), value(DefaultMenuBackgroundColor)));
+    menuBorderWidth: fallback(value(MenuBorderWidth), value(DefaultMenuBorderWidth));
+    menuBorderColor: color(fallback(value(MenuBorderColor), value(DefaultMenuBorderColor)));
+}
+
+.root {
+    -st-states: mobile, alignment(enum(left, center, right));
+    padding: 12px 24px;
+    max-width: 320px;
+    background-color: value(menuBackgroundColor);
+    border-width: value(menuBorderWidth);
+    border-color: value(menuBorderColor);
+}
+
+.root:mobile {
+    width: 100%;
+}
+
+.root:alignment(left) {
+    text-align: left;
```

Comment:
As @giladsegal wrote, this isn't working as excepted.
However, using `text-align` is fine in this case, since this is the component's alignment, and not the text-direction.
Try to solve why this isn't working correctly.

Reply:
I think you need to import the `ActionsMenuItem.st.css` file here, and pass the alignment to the component.